backend/utils/models/base_model.py

```
@@ -1,9 +1,53 @@
 
+    objects = BaseModelManager()
+
     class Meta:
         abstract = True
+
+    def save(self, *args, update_fields=None, **kwargs):
+        # Django only fires auto_now for fields listed in update_fields, so a
+        # partial save() silently drops the modified_at bump. Auto-include it
+        # whenever the caller restricts update_fields.
+        if update_fields is not None and "modified_at" not in update_fields:
+            update_fields = list(update_fields) + ["modified_at"]
```

Ruff (0.15.10)

[warning] Consider `[*list(update_fields), "modified_at"]` instead of concatenation

Replace with `[*list(update_fields), "modified_at"]`

(RUF005)